3party/jansson/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+cmake_minimum_required(VERSION 3.2)
+
+project(jansson C CXX)
+
+include_directories(src ../../)
+
+add_compile_options(
+    "-Wall"
+    "-Wno-unused-function"
+)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
No, otherwise I get this error:
```[  8%] Building C object 3party/jansson/CMakeFiles/jansson.dir/src/dump.c.o
error: invalid argument '-std=c++11' not allowed with 'C/ObjC'```